starknet_py/constants.py

```diff
@@ -10,3 +12,5 @@
 RPC_CONTRACT_NOT_FOUND_ERROR = 20
 RPC_INVALID_MESSAGE_SELECTOR_ERROR = 21
 RPC_CLASS_HASH_NOT_FOUND_ERROR = 28
+
+ROOT_PATH = Path(__file__).parent
```

Comment:
Isn't the result of this dependent on how `constants.py` is imported?

Reply:
We could add an **assert** below
```
ROOT_PATH = Path(__file__).parent
assert ROOT_PATH.name == "starknet_py"
```

Reply:
> Isn't the result of this depends on how constants.py is imported?

It is but constants.py is in the top-level directory. Should I add an `assert` as @drknzz sad?

Reply:
Isn't this 
```py
ROOT_PATH = Path(__file__).parent
assert ROOT_PATH.name == "starknet_py"\
```
pretty much equal to just hardocoding the path (i.e. not using `__file__` and parent) at all?

Reply:
You're right. I will rewrite that

Reply:
The problem with hardcoding is that we can hardcode only the part of the path ("starknet_py"). Path(__file__) takes the entire path from the Home directory which is more useful

Reply:
> Isn't this
> 
> ```python
> ROOT_PATH = Path(__file__).parent
> assert ROOT_PATH.name == "starknet_py"\
> ```
> 
> pretty much equal to just hardocoding the path (i.e. not using `__file__` and parent) at all?

So just
```
ROOT_PATH = "starknet_py"
```
?

Would this be any useful? Would this work here?
```
file_path = ROOT_PATH / "tests/e2e/docs/code_examples" / file_name
```

Reply:
Nope, it wouldn't. (tested that)